client/src/api/datasets.ts

```diff
@@ -67,7 +67,7 @@ export async function purgeDataset(datasetId: string) {
 }
 
 const datasetCopy = fetcher.path("/api/histories/{history_id}/contents/{type}s").method("post").create();
-type HistoryContentsArgs = FetchArgType<typeof datasetCopy>;
+export type HistoryContentsArgs = FetchArgType<typeof datasetCopy>;
```

Comment:
Just a heads up, this is going to create a conflict when merging forward since `FetchArgType` no longer exists in `dev`.
What I recommend is to leave this line as it was and then add at the bottom of this file the 2 types that you really need in  `HistoryPanel.vue`.

Something like this:
```ts
export type HistoryContentType = components["schemas"]["HistoryContentType"];
export type HistoryContentSource = components["schemas"]["HistoryContentSource"];
```

Reply:
Makes sense, thank you! I will try to keep in mind such cases where something could create conflicts merging forward 👍